tests/unit_tests/recipes/llama/test_llama_recipes.py

```diff
@@ -0,0 +1,107 @@
+import importlib
```

Comment:
could you add comments for how this is expected to work? this seems much less precise than the existing configs we could validate against. 

what was the issue with keeping the existing recipe test configs to validate defaults? is it the HF config download we want to avoid for the tests here?